notte/llms/config/endpoints.csv

```diff
@@ -0,0 +1,6 @@
+provider,model,rpm,tpm,rph,tph,rpd,tpd

```

Comment:
I think this should be in the `llamux` library not in notte. The rate limits are the same for all users/projects.

Reply:
It's up to the users to define the endpoints they want to use; +the rate limits depends on tiers and stuff like that. There is too much variance for it to be handled by `llamux` for very small effort on user side (us); at least for now.

Reply:
@andreakiro need answer here

Reply:
https://github.com/nottelabs/notte/issues/28